app/views/volunteers/_form.html.slim

```diff
@@ -58,9 +58,14 @@
     = f.input :skills, label: t('your_skills')
     = f.input :interests, label: t('your_interests')
 
-.row
-  .col-xs-12
-    = f.input :state, as: :hidden
+- if current_user.present? && policy(Volunteer).destroy?
```

Comment:
`current_user.present?` can be avoided by doing this
```ruby
def destroy?
  user && user.superadmin?
end
```
in the `VolunteerPolicy`

so you only need
```ruby
if policy(Volunteer).destroy?
```